packages/shared/package.json

```diff
@@ -20,9 +20,9 @@
     "graphql": "^15.5.0",
     "graphql-request": "^3.4.0",
     "idb-keyval": "^5.0.6",
-    "next": "^10.2.3",
-    "react": "^17.0.5",
-    "react-dom": "^17.0.5",
+    "next": "^11.1.3",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
```

Comment:
Why did we have to downgrade our react packages?

Reply:
I think this might be a upgrade that went wrong as 17.0.5 doesn't even exist.

![Screenshot 2022-01-20 at 09 25 17](https://user-images.githubusercontent.com/554874/150292414-c72ca82a-db2f-4736-b423-f4e35deaa2ed.png)

Reply:
But upgraded it to the latest version now